openfecwebapp/templates/partials/committee-totals-house-senate.html

```diff
@@ -30,7 +30,6 @@
      (totals.0.refunded_individual_contributions, 'Individual refunds'),
      (totals.0.refunded_political_party_committee_contributions, 'Political party refunds'),
      (totals.0.refunded_other_political_committee_contributions, 'Other committee refunds'),
-     (totals.0.other_disbursements, 'Other disbursements'),
```

Comment:
Looking at the current committee summaries and I think we want to keep the first `other disbursements` and switch out the duplicative `other disbursements` at the end with `total disbursements`.

http://www.fec.gov/fecviewer/CandidateCommitteeDetail.do (look up a house or senate member)